zipline/modelling/term.py

```diff
@@ -0,0 +1,294 @@
+"""
+Base class for Filters, Factors and Classifiers
+"""
+from numpy import (
+    empty,
+    float64,
+    full,
+    nan,
+)
+
+from zipline.errors import (
+    InputTermNotAtomic,
+    TermInputsNotSpecified,
+    WindowLengthNotPositive,
+    WindowLengthNotSpecified,
+)
+from zipline.utils.lazyval import lazyval
```

Comment:
Should these be relative imports? https://docs.python.org/3/reference/import.html#submodules